R/pkg/R/SQLContext.R

```diff
@@ -544,12 +544,15 @@ sql <- function(x, ...) {
   dispatchFunc("sql(sqlQuery)", x, ...)
 }
 
-#' Create a SparkDataFrame from a SparkSQL Table
+#' Create a SparkDataFrame from a SparkSQL table or temporary view
```

Comment:
`table or view`

Here, actually, it includes both temporary views or persistent views.